auth-jwt/auth-jwt-java-sbt/README.md

```diff
@@ -0,0 +1,48 @@
+# Lagom Recipe: How authenticate/authorize by JWT
+
+This recipe demonstrates, how you can use [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token) 
+for protection your service endpoints. It used library [PAC4J](https://www.pac4j.org/) 
+and her [module](https://github.com/pac4j/lagom-pac4j) of integration with Lagom. 
+
+## Testing the recipe
+
+#### unit tests
+
+You can test this recipe using the provided tests:
+
+```bash
+sbt test
+```
+
+#### manual tests
+
+You can also test this recipe manually using 2 separate terminals.
+
+On one terminal start the service:
+
+```bash
+sbt runAll
+```
+
+On a separate terminal, use `curl` to send:
+
+* anonymous request
+```bash
+curl http://localhost:9000/authenticate
```

Comment:
I tried this out and got an 404.

The path is missing `/jwt/`

```
curl http://localhost:9000/jwt/authenticate
```

Reply:
Sorry 😞
Fix by removed `/jwt` prefix on the service descriptor.